.gitignore

```diff
@@ -2,6 +2,7 @@ lib/
 module/
 types/
 storybook-static/
+.idea/
```

Comment:
I'm using WebStorm, so I'd like to add this line if possible.
Since https://www.gitignore.io/api/JetBrains allows us to add some WebStorm-specific files, I would recommend not to use https://www.gitignore.io/api/JetBrains.